etc/k8s/images/kernelpatch-node.Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM debian
```

Comment:
Let's consolidate all dockerfiles in one file to create a single image. Move the runtime command to the daemon set..

Reply:
Great idea! Fixed in the latest commit.